lib/event_class.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+class Event
+attr_reader   :name
+              :ages
+
+  def initialize(curling, ages)
+    @curling = curling
+    @ages = [24, 30, 18, 20, 41]
+  end
+
+  def max_age
+    @ages.sort.last
+  end
+
+  def min_age
+    @ages.sort.first
+  end
+
+  def average_age
```

Comment:
For the average calculation, you won't necessarily need to use `.each`.  You could do something like this:

```ruby
def average_age
  (@ages.sum / @ages.count.to_f).round(2)
end
```